src/jit/codegenxarch.cpp

```diff
@@ -1293,8 +1293,26 @@ void CodeGen::genCodeForDivMod(GenTreeOp* treeNode)
     var_types  targetType = treeNode->TypeGet();
     emitter*   emit       = getEmitter();
 
-    // dividend is not contained.
-    assert(!dividend->isContained());
+#ifdef _TARGET_X86_
+    bool dividendIsLong = varTypeIsLong(dividend->TypeGet());
```

Comment:
Since you're touching this function, it would be good to add a standard function header